### PR TITLE
Storage resource annotations

### PR DESCRIPTION
**Pull request by okolo**

### CODE REVIEW

File: ontology.ttl [hunk 1]
```diff
@@ -576,11 +525,6 @@ foaf:logo rdf:type owl:AnnotationProperty .
                 rdfs:subClassOf :DataProduct .
 
 
-###  http://odahub.io/ontology#POSIXPath
-:POSIXPath rdf:type owl:Class ;
-           rdfs:subClassOf :FileReference .
-
-
```
Comment (dsavchenko) [inline ontology.ttl:583]: Why this and other removals? Did you merge `main` properly?

**Review by dsavchenko — CHANGES_REQUESTED** [review]

File: ontology.ttl (hunk 1 above)
Comment (okolo) [reply ontology.ttl]: Indeed, something went wrong during merge. I have recovered the changes from main branch

File: ontology.ttl [hunk 2]
```diff
@@ -161,6 +166,27 @@ foaf:logo rdf:type owl:AnnotationProperty .
...
+
+###  http://odahub.io/ontology#usesRequiredResource
+:usesRequiredResource rdf:type owl:ObjectProperty ;
+                      rdfs:subPropertyOf :usesResource .
+
+
+###  http://odahub.io/ontology#usesResource
+:usesResource rdf:type owl:ObjectProperty ;
+              rdfs:subPropertyOf owl:topObjectProperty ;
+              rdfs:domain :Workflow .
```
Comment (dsavchenko) [inline ontology.ttl:187]:
```suggestion
              rdfs:domain :Workflow ;
              rdfs:range :Resource.
```

File: ontology.ttl [hunk 3]
```diff
@@ -93,6 +93,11 @@ foaf:logo rdf:type owl:AnnotationProperty .
 #    Object Properties
 #################################################################
 
+###  http://odahub.io/ontology#accessMode
+:accessMode rdf:type owl:ObjectProperty ;
+            rdfs:domain :StorageResource .
```
Comment (dsavchenko) [inline ontology.ttl:98]: Would be good to define `rdfs:range`.

May it be used in the definition of `:CRBeamS3` properties?

File: ontology.ttl [hunk 4]
```diff
@@ -161,6 +166,27 @@ foaf:logo rdf:type owl:AnnotationProperty .
          rdfs:range :Workflow .
 
 
+###  http://odahub.io/ontology#resourceBindingEnvVarName
+:resourceBindingEnvVarName rdf:type owl:ObjectProperty ;
+                           rdfs:domain :S3 .
```
Comment (dsavchenko) [inline ontology.ttl:171]: Only S3? May it be applicable to the whole StorageResource?

File: ontology.ttl [hunk 5]
```diff
@@ -161,6 +166,27 @@ foaf:logo rdf:type owl:AnnotationProperty .
          rdfs:range :Workflow .
 
 
+###  http://odahub.io/ontology#resourceBindingEnvVarName
+:resourceBindingEnvVarName rdf:type owl:ObjectProperty ;
```
Comment (dsavchenko) [inline ontology.ttl:170]: Isn't it a DataProperty? Its value is a literal, not Class

**Review by dsavchenko — CHANGES_REQUESTED** [review]

File: ontology.ttl (hunk 4 above)
Comment (okolo) [reply ontology.ttl]: Resolved in the latest commit

File: ontology.ttl (hunk 3 above)
Comment (okolo) [reply ontology.ttl:98]: What values are we going to use? String constants "read","read_write","read_append"?
What about type, should it be also DataProperty?

Comment (dsavchenko) [reply ontology.ttl:98]: If string values, then yes, it's a DataProperty and there is no need in range then

**Review by dsavchenko — APPROVED** [review]

File: ontology.ttl [hunk 6]
```diff
@@ -1040,6 +1092,26 @@ Subclasses may have other ways to express the content.""" ;
 #    Individuals
 #################################################################
 
+###  http://odahub.io/ontology#CRBeamS3
+:CRBeamS3 rdf:type owl:NamedIndividual ,
```
Comment (volodymyrss) [inline ontology.ttl:1096]: it's not a `StorageResource`?

**Review by volodymyrss — APPROVED** [review]

**Review by volodymyrss — APPROVED** [review]
Ok thanks!

**Pull request merged**